apps/antalmanac/src/stores/SettingsStore.ts

```diff
@@ -23,9 +25,19 @@ export const useThemeStore = create<ThemeStore>((set) => {
             ? 'dark'
             : 'light';
 
+    const isDark =
+        themeSetting !== 'system'
+            ? themeSetting === 'dark'
+                ? true
+                : false
+            : window.matchMedia('(prefers-color-scheme: dark)').matches
+            ? true
+            : false;
```

Comment:
This code chunk is repeated below, so it should be extracted to a function in this file named something like `computeTheme()` or `themeShouldBeDark()`.

There are also serious problems about the way your wrote the logic here.
1) This does not tell you how the condition works if you don't already know. In a function, you can name each condition and combine them to make readable logical sentences. In a different format, you can watch 20 seconds of [this video](https://youtu.be/Bf7vDBBOBUA?si=p_G2cEH9-xhZpmRj&t=40), which I'd highly recommend.
2) The two innermost ternary operations are completely redundant. `themeSetting === 'dark' ? true : false` is equivalent to `themeSetting`. Same with the `matchMedia`. Simplifying this could justify this inline condition *if it only appears once*. Since it doesn't it should be extracted in any case.

I know this seems trivial, but it's critical to write clean, readable code to avoid logical mistakes and save our time. (As trivia, it's at least important enough for Professor Klefstad to mark you down.)